tvb_documentation/demos/simulate_for_mouse.ipynb

```diff
@@ -63,14 +63,35 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {
     "tags": []
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Pls clean the output cells before you submit notebooks, or else we end up with unreadable diffs